src/sqlittaaja/__init__.py

```diff
@@ -3,20 +3,25 @@
 from sqlittaaja.checker import check_exercises
 from sqlittaaja.diff_check import compute_similarity
 from sqlittaaja.extractor import extract
+from sqlittaaja.gui import start_gui
+import sys
 import tomllib
 
 
 def main():
-    args = read_args()
+    if "-c" not in sys.argv and "--config" not in sys.argv:
```

Comment:
Please don't do this. We have the `read_args()` for a reason. Check that if `--config` was specified or not. Still, it would be better to add argument `--gui` or something to force the use of a GUI. Otherwise you could check `sys.stdin.isatty()` to check if the program was launched from a terminal or not.